packages/core/core/src/utils/transform-content-types-to-models.ts

```diff
@@ -317,6 +317,17 @@ export const transformContentTypesToModels = (
       },
     };
 
+    // Add indexes to model
+    if (contentType.modelType === 'contentType') {
+      model.indexes = [
+        {
+          name: identifiers.getIndexName([contentType.collectionName, 'documents']),
+          // Filter attributes that are not in the schema
+          columns: ['documentId', 'locale', 'publishedAt'].filter((n) => model.attributes[n]).map( _.snakeCase),
```

Comment:
@innerdvations ccould we check this together at some point? I would like to know if I should use something else here, because this seems off.

I am creating the database indexes (which should be snakecased) but filtering by model attributes (which are camelcased)